components/salesforce_rest_api/README.md

```diff
@@ -21,10 +29,7 @@ Here is a step-by-step on how to do this:
 4. Create a new permission set, give it a label, API name, and description. Example:
 **Label**: Pipedream API Access
 **API Name**: Pipedream
-**Description**: Adds a set of permissions required for Pipedream sources:
-- Apex REST Services
-- API Enabled
-- Author Apex
+**Description**: Adds a set of permissions required for Pipedream. 
```

Comment:
**-Minor-**
The example renders like this for me on https://pipedream.com/apps/salesforce-rest-api:
<img width="828" alt="f06f662" src="https://github.com/PipedreamHQ/pipedream/assets/19861096/22386714-92dc-402b-97e7-281eec50372e">

I think you can get it to render on separate lines by using a [line break element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) `<br>` after "Example:".

This markdown
```md
4. Create a new permission set, give it a label, API name, and description. Example:<br>**Label**: Pipedream API Access
**API Name**: Pipedream
**Description**: Adds a set of permissions required for Pipedream. 
```
renders as:

4. Create a new permission set, give it a label, API name, and description. Example:<br>**Label**: Pipedream API Access
**API Name**: Pipedream
**Description**: Adds a set of permissions required for Pipedream.